WordPressVIPMinimum/Tests/Filters/RestrictedFilterUnitTest.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+
+add_filter( 'test_filter', 'good_example_function' ); 	// this one shouldn't trigger anything
+
+add_filter( 'upload_mimes', 'bad_example_function' );
```

Comment:
I'd like to see more tests please:

 - combinations of whitespace / no-whitespace before and after the filter name arg i.e. `add_filter('upload_mimes' , 'foo' );`
 - string with upper case characters (`'UPLoad_MimEs'`)
 - string with double quotes (`"upload_mimes"`)
 - strings with concatenation (`'upload_' . 'mimes'`) 
 - strings with compounded strings/vars
   - `$x = 'd_m'; add_filter( 'uploa' . $x . 'imes', 'foo')`
   - `$x = 'd_m'; add_filter( "uploa{$x}imes", 'foo')`
 - `add_filter()` calls with anonymous function callbacks or other callables.
 - `add_filter()` called dynamically (may be overkill?)
   - `$x = 'add_filter'; $x( 'upload_mimes', 'foo' );`
 - etc.

See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/Tests/NamingConventions/ValidHookNameUnitTest.1.inc and sibling files for other ideas of ways folks could try and bypass the check by mangling the filter name string.

Your code may already cope with some of those (like double-quotes), but adding them as tests just proves that for the future.

Also, keep a clear linespace on the last line (adjust your editor to do this, since the repo doesn't yet have an `.editorconfig` file).